### PR TITLE
Feat: CartPage

### PR DESCRIPTION
**Pull request by yashpatel1797**
Added Cartpage
[live link](https://confident-wozniak-7f4f57.netlify.app/page/cart.html)

### CODE REVIEW

**Comment by parthpandyappp** [comment]
LGTM!
Suggestion: Color contrast of UI can be improved once the layout and alignment of components are done for the app.

**Comment by yashpatel1797** [comment]
> LGTM! Suggestion: Color contrast of UI can be improved once the layout and alignment of components are done for the app.

yes, once the design will complete I will update color for site.

**Comment by parthpandyappp** [comment]
> > LGTM! Suggestion: Color contrast of UI can be improved once the layout and alignment of components are done for the app.
> 
> yes, once the design will complete I will update color for site.

Yup, makes sense. Fine :+1:

**Review by parthpandyappp — APPROVED** [review]

File: style/cart.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+.cart-checkout > div {
+  display: flex;
+  justify-content: space-between;
+  margin: 1rem 0rem;
+}
+.image-container {
+  width: 25rem;
+}
+.text-container-title {
+  padding: 1rem 0rem 1rem 0rem;
```
Comment (bishnoimukesh) [inline style/cart.css]: use shorthand

**Review by bishnoimukesh — COMMENTED** [review]
suggested change rest looks good to merge

**Pull request merged**